migration_toolkit_for_containers/index.adoc

```diff
@@ -0,0 +1,79 @@
+[id="migration-toolkit-for-containers-overview"]
+= Migration toolkit for containers overview
+include::modules/common-attributes.adoc[]
+:context: migration-toolkit-for-containers-overview
+
+toc::[]
+
+You can migrate stateful application workloads between {product-title} 4 clusters at the granularity of a namespace by using the Migration Toolkit for Containers (MTC). To learn more about MTC see xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+[discrete]
+[id='mtc-overview-install-mtc']
+== MTC installation
+xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[MTC installation]: You can install the Migration Toolkit for Containers (MTC) on {product-title} 4. After installing MTC, you must configure an object storage to use as a replication repository. You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You can check the following items:
```

Comment:
This is out of order. 
Remove first sentence.
You install MTC Operator on 4.6+ with OLM. 
You install legacy MTC Operator on 4.5-, manually.
Then you add the replication repository.

Also, why would the user want to check any of the following items? They are steps:

1. Install Operator.
2. Configure replication repository.